data-processing/perl/setup.sh

```diff
@@ -0,0 +1,9 @@
+# Fetch sources for AI2 clone of LaTeXML.
+git clone https://github.com/allenai/latexml latexml
```

Comment:
I was just wondering if we'd ever have a breaking change merge to our fork of latexml that then might then break the processor until it was updated?

Reply:
This is a great point. It could certainly happen.

I wonder what types of merge workflows could prevent this from happening. Maybe an explicit tag of the LaTeXML repo is cloned, so that there is no unexpected changes in the behavior of the processor as the LaTeXML branch updates?

Reply:
yup explicitly pinning the tag/commit of the latexXML repo here at cloning time is probably good enough 👍

Reply:
good thinking!